EIPS/eip-x.md

```diff
@@ -25,61 +25,69 @@ The ERC-4626 Tokenized Vaults standard has helped to make yield-bearing tokens m
 | ---------:|:-------------------------------- | ------------------------------------------------------------------------------------------------------------------------:|
 |   Pending | requestDeposit(assets, operator) |                               asset.transferFrom(msg.sender, vault, assets)<br>pendingDepositRequest[operator] += assets |
-| Claimable |                                  | <i>Internal request fulfillment</i><br>pendingDepositRequest[msg.sender] -= assets<br>maxDeposit[operator] += assets<br> |
+| Claimable |                                  | <i>Internal Request fulfillment</i><br>pendingDepositRequest[msg.sender] -= assets<br>maxDeposit[operator] += assets<br> |
 |   Claimed | deposit(assets, receiver)        |                                                  maxDeposit[msg.sender] -= assets<br>vault.balanceOf[receiver] += shares |
 
-An important vault inequality is that following a Request(s), the cumulative requested quantity MUST be more than `pendingDepositRequest + maxDeposit - claimed`. The inequality may come from fees or other state transitions outside implemented by vault logic such as cancellation of a Request, otherwise this would be a strict equality.
+An important Vault inequality is that following a Request(s), the cumulative requested quantity MUST be more than `pendingDepositRequest + maxDeposit - claimed`. The inequality may come from fees or other state transitions outside implemented by Vault logic such as cancellation of a Request, otherwise this would be a strict equality.
 
-Requests MUST NOT skip or otherwise short-circuit the claim step. In other words, to initiate and claim a request, a user MUST call both request* and the corresponding claim function separately, even in the same block.
+Requests MUST NOT skip or otherwise short-circuit the Claim state. In other words, to initiate and claim a Request, a user MUST call both request* and the corresponding Claim function separately, even in the same block.
 
-For asynchronous request vaults, the exchange rate between shares and assets including fees and yield is up to the vault implementation. In other words, pending redemption requests MAY NOT be yield bearing and MAY NOT have a fixed exchange rate.
+For asynchronous Vaults, the exchange rate between `shares` and `assets` including fees and yield is up to the Vault implementation. In other words, pending redemption Requests MAY NOT be yield bearing and MAY NOT have a fixed exchange rate.
+
+The view function relationship is summarized in the below table where the Request input amount is 1000 and pending/max view function input is the `operator`:
+
+| **State**   | **Pending View Output** | **Max View Output** |
+|-------------|-------------------------|--------------------|
+| Pending     | 1000                    | 0                  |
+| Claimable   | 0                       | 1000               |
+| Claimed     | 0                       | 0                  |
```

Comment:
think this table could use a clarification that the max view function being 1000 is contingent on the protocol fulfilling that pending request

noticed that we got rid of the suggested invariant as well, but 

`pendingDepositRequest = pendingViewAmount - maxView` should still hold right? assuming no cancelations or fees

Reply:
Related to the comment above, I think this table adds more confusion than explanation. The other table also already shows how the pending and max view functions change outputs.